### PR TITLE
Adding a new Java webframework-Rapidoid

### PR DESCRIPTION
**Pull request by anujbahuguna**

### CODE REVIEW

**Comment by akullpp** [comment]
Looks good, please fix the issues and we will add it.

File: README.md [hunk 1]
```diff
@@ -629,6 +629,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Pippo](http://www.pippo.ro/) - Small, highly modularized Sinatra-like framework.
 * [Play](https://www.playframework.com/) - Uses convention over configuration, hot code reloading and display of errors in the browser.
 * [PrimeFaces](http://primefaces.org/) - JSF framework which has a free and a commercial version with support. Provides several frontend components.
+* [Rapidoid](http://www.rapidoid.org/) - An Open source (Apache license), Simple, Secure and Extremely fast Web Framework for Java.
```
Comment (akullpp) [inline README.md]: The consensus was to write `open-source`, also you don't need to mention the license. Also the description is very much generic, could you please fix it and be as concise as possible?

Comment (anujbahuguna) [inline README.md]: Sure, have modified the description based on above feedback to 
"A Simple, Secure and Extremely fast Web Framework consisting of embedded HTTP server, GUI components and Dependency Injection."

File: README.md [hunk 2]
```diff
@@ -629,6 +629,7 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Pippo](http://www.pippo.ro/) - Small, highly modularized Sinatra-like framework.
 * [Play](https://www.playframework.com/) - Uses convention over configuration, hot code reloading and display of errors in the browser.
 * [PrimeFaces](http://primefaces.org/) - JSF framework which has a free and a commercial version with support. Provides several frontend components.
+* [Rapidoid](http://www.rapidoid.org/) - A Simple, Secure and Extremely fast Web Framework consisting of embedded HTTP server, GUI components and Dependency Injection.
```
Comment (akullpp) [inline README.md]: Spelling corrected:

` A Simple, secure and extremely fast framework consisting of embedded HTTP server, GUI components and dependency injection.`

**Pull request closed without merging**